screens/SendSetupScreen.js

```diff
@@ -295,7 +282,134 @@ const SendSetupScreen = ({
     }
   };
 
+  const handleDeepLink = ({
+    address,
+    amount,
+    tokenAmount,
+    label,
+    tokenId,
+    symbol
+  }): DeepLinkParams => {
+    const type = getType(address);
+
+    if (typeof type !== "string") {
+      setErrors(["Invalid Address"]);
+    }
+    if (type === "cashaddr") {
+      return prefillBCH({ address, amount, label });
+    }
+    if (type === "slpaddr") {
+      prefillSLP({
+        address,
+        amount,
+        tokenAmount,
+        label,
+        tokenId,
+        symbol
+      });
+    }
+  };
+
+  const prefillSLP = ({
+    address,
+    amount,
+    tokenAmount,
+    label,
+    tokenId,
+    symbol
+  }): DeepLinkParams => {
+    const hasTokenAmount = address !== undefined && tokenAmount !== undefined;
+
+    if (tokenId === undefined) {
+      setErrors(["Missing Token Type"]);
+      redirectHome(navigation);
+      return null;
+    }
+
+    if (hasTokenAmount) {
+      try {
+        setSendAmountCrypto(tokenAmount);
+        // const numericalAmount = parseAmount(tokenAmount);
+        setAmountType("crypto");
+        setSendAmount(tokenAmount);
+      } catch (error) {
+        setErrors(["Invalid Amount"]);
+        redirectHome(navigation);
+      }
+    }
+    setToAddress(address);
+  };
+
+  const prefillBCH = ({
+    address,
+    amount,
+    tokenAmount,
+    label,
+    tokenId,
+    symbol
+  }): DeepLinkParams => {
+    const hasAmount = address !== undefined && amount !== undefined;
+    navigation.state.params.symbol = "BCH";
+    if (hasAmount) {
+      try {
+        setSendAmountCrypto(amount);
+        const numericalAmount = parseAmount(amount);
+        setAmountType("crypto");
+        setSendAmount(amount);
+        setSendAmountFiat(
+          fiatRate
+            ? (fiatRate * (numericalAmount || 0)).toFixed(
+                currencyDecimalMap[fiatCurrency]
+              )
+            : 0
+        );
+      } catch (error) {
+        setErrors(["invalid amount"]);
+      }
+    }
+
+    setToAddress(address);
+  };
+
+  const parseQr = (qrData: string, tokensById: object) => {
```

Comment:
change the type `object` to `any`, or enter it's actual type.

object with a lowercase `o` isn't a think, and `Object` with an uppercase `O` is being deprecated for `any`
https://flow.org/en/docs/types/objects/#toc-object-type